### PR TITLE
Add agent specific configurations

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: modules/apim_master/templates/carbon-home/repository/conf/carbon.xml.erb [hunk 1]
```diff
@@ -44,12 +44,12 @@
        This is will become part of the End Point Reference of the
        services deployed on this server instance.
     -->
-    <!--HostName><%= @hostname %></HostName-->
+    <HostNam></HostNam>
```
Comment (VimukthiPerera) [inline modules/apim_master/templates/carbon-home/repository/conf/carbon.xml.erb]: Should be `HostName`

Comment (VimukthiPerera) [reply modules/apim_master/templates/carbon-home/repository/conf/carbon.xml.erb]: Also, why is this empty? Shouldn't this be parameterized?

File: modules/apim_km_master/templates/carbon-home/repository/conf/carbon.xml.erb [hunk 2]
```diff
@@ -44,12 +44,12 @@
        This is will become part of the End Point Reference of the
        services deployed on this server instance.
     -->
-    <!--HostName><%= @hostname %></HostName-->
+    <!--<HostName><%= @hostname %></HostName>-->
```
Comment (VimukthiPerera) [inline modules/apim_km_master/templates/carbon-home/repository/conf/carbon.xml.erb:47]: Are we keeping this commented out?

Comment (Savidude) [reply modules/apim_km_master/templates/carbon-home/repository/conf/carbon.xml.erb:47]: The GA pack has it commented out

**Review by ctienshi — APPROVED** [review]

**Pull request merged**